client/app/components/_languageChooser/languageChooser.spec.js

```diff
@@ -0,0 +1,41 @@
+import LanguageChooserModule from "./languageChooser";
+
+const { module } = angular.mock;
+
+describe("LanguageChooser", () => {
+  beforeEach(module(LanguageChooserModule));
+
+  describe("Module", () => {
+    it("is named languageChooser", () => {
+      expect(LanguageChooserModule).to.equal("languageChooser");
+    });
+  });
+
+  describe("Controller", () => {
+    let $componentController;
+    beforeEach(inject(($injector) => {
+      $componentController = $injector.get("$componentController");
+    }));
+
+    it("should exist", () => {
+      let ctrl = $componentController("languageChooser", {});
+      expect(ctrl).to.exist;
+    });
+
+    context("stubbed translate", () => {
```

Comment:
I had another test before that actually calls use(), but I couldn't make it work (even with reading the documentation). So I deleted it and was too lazy to remove this separated space for the stub. Will do..